### PR TITLE
Replace symlink OS folders with actual folders, remove unused kickstart/preseed configs

### PR DESCRIPTION
**Pull request by abhay-krishna**
This PR extends some of the work done in #1427 to replace the OS folder symlinks with the actual folders and content to isolate changes to a specific OS or OS version and avoid breaking something unintentionally. It also removes unused autoinstall config files for builds whose Packer config doesn't exist.

### CODE REVIEW

**Comment by AverageMarcus** [comment]
Wow the GitHub PR view really makes this hard to follow doesn't it? 😆 

I took a look at the branch directly and looks good to me :)

/lgtm
/assign @mboersma

**Review by mboersma — APPROVED** [review]
/lgtm
/approve
